src/BlazorUI/Bit.BlazorUI.Extras/Styles/Fluent2/tokens.fluent2.scss

```
@@ -116,6 +117,12 @@
     --bit-siz-slider-thumb-md: calc(var(--bit-spa-scaling-factor) * var(--bit-layout-density-scale) * 2); //16px
     --bit-siz-slider-thumb-lg: calc(var(--bit-spa-scaling-factor) * var(--bit-layout-density-scale) * 2.5); //20px
 
+    /*---- Opacity ----*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)